ghcjs-ffi/Miso/FFI.hs

```diff
@@ -88,21 +89,28 @@ objectToJSVal = pure . jsval
 ghcjsPure :: a -> JSM a
 ghcjsPure = pure
 
+-- | Forces execution of pending asyncronous code
```

Comment:
Not sure if this is right (just copied if over from jsaddle docs). Does it mean that this only have an effect when running with jsaddle?